$OpenBSD: patch-configure,v 1.4 2017/09/16 22:52:40 jeremy Exp $

Fix so name, checking for DOT and DOXYGEN.

Override the arch setting to remove OpenBSD version from it,
so ports don't have to be bumped when OpenBSD version changes.

--- configure.orig	Wed Mar 30 02:48:57 2016
+++ configure	Tue Jul 12 11:32:17 2016
@@ -20971,7 +20971,7 @@ esac
   openbsd*|mirbsd*) :
 
 	SOLIBS='$(LIBS)'
-	LIBRUBY_SO='lib$(RUBY_SO_NAME).so.$(MAJOR).'`expr ${MINOR} \* 10 + ${TEENY}`
+	LIBRUBY_SO='lib$(RUBY_SO_NAME).so.'${LIBruby21_VERSION}
 	 ;; #(
   solaris*) :
 
@@ -22215,7 +22215,10 @@ _ACEOF
 _ACEOF
 
 else
-    arch="${target_cpu}-${target_os}"
+    case "$target_os" in
+      openbsd*) arch="$target_cpu-openbsd" ;;
+      *) arch="${target_cpu}-${target_os}" ;;
+    esac
     cat >>confdefs.h <<_ACEOF
 #define RUBY_PLATFORM "${arch}"
 _ACEOF
